_data/simple-icons.json

```diff
@@ -12219,6 +12219,11 @@
             "hex": "2F2E8B",
             "source": "https://strapi.io/newsroom"
         },
+        {
+            "title": "Strato",
+            "hex": "FF8800",
+            "source": "https://www.strato.de"
```

Comment:
```suggestion
            "source": "https://www.strato.de/press",
            "guidelines": "https://www.strato.de/press"
```

The forbidden line about requiring permission, would you mind reaching out @PratyushJayachandran?

Reply:
Hah okay. I'll do.

Reply:
Awaiting reply from Strato

Reply:
No reply 😴 
Shall we just close this?

Reply:
Might be best. Could you also comment on the issue and close it as a won't add?